freqtrade/arguments.py

```diff
@@ -364,23 +367,24 @@ def testdata_dl_options(self) -> None:
         """
         self.parser.add_argument(
             '--pairs-file',
-            help='File containing a list of pairs to download',
+            help='File containing a list of pairs to download.',
             dest='pairs_file',
             default=None,
             metavar='PATH',
         )
 
         self.parser.add_argument(
             '--export',
-            help='Export files to given dir',
+            help='Export files to given dir.',
             dest='export',
             default=None,
             metavar='PATH',
         )
 
         self.parser.add_argument(
             '-c', '--config',
-            help='specify configuration file, used for additional exchange parameters',
+            help='Specify configuration file, used for additional exchange parameters. '
+                 'Multiple --config options may be used.',
```

Comment:
I think you will want to align this to line 78 (which lacks the comment, but has the action correct)

Reply:
> I think you will want to align this to line 78 (which lacks the comment, but has the action correct)

Did not get it, sorry. Can you explain or simply change by yourself?

Reply:
sorry for the confusion.
the property "--config" is defined twice - once in this line, and once in the same file, in line 78.

the "help"-comment was changed here - but the implementation on the other occurence.

Reply:
Done. See new commit below: https://github.com/freqtrade/freqtrade/pull/1571/commits/87c82dea3d93a8321a6594e024e48ad3b30c5b17